modules/gdscript/tests/scripts/analyzer/errors/abstract_script_instantiate.out

```diff
@@ -0,0 +1,2 @@
+GDTEST_ANALYZER_ERROR
+Native class CanvasItem is abstract and can't be instantiated.
```

Comment:
I think this message might be confusing in this case. You need to be aware that `B` inherits from `CanvasItem` otherwise you'll be wondering where this comes from.

IMO it should be something like: `Class "B" is based on the abstract class "CanvasItem" and can't be instantiated.`

Reply:
I changed wording of the error. Unfortunately there is no helper method to easily get presentable name for a class. So for now without it in the error, but it is clearer. I'll resolve this, if there is a simple method let me know. (`fqcn` contained full system path in tests at least and not every class has a name.)

Reply:
Can't you use the ClassNode->fqcn (fully qualified class name) or something like that?

Reply:
> `fqcn` contained full system path

Reply:
Yes! I took that and used `.get_file()` on it!

If there's a `class_name`, I believe it gives you the class name without any path because it's a global name. If there's no `class_name`, then you use `get_file()` and get the filename containing the "unnamed" class with `.gd`.

Reply:
Done.